std/algorithm/searching.d

```diff
@@ -2777,23 +2777,30 @@ smallest element and with the same ending as $(D range). The function
 can actually be used for finding the maximum or any other ordering
 predicate (that's why $(D maxPos) is not provided).
  */
-Range minPos(alias pred = "a < b", Range)(Range range)
+auto minPos(alias pred = "a < b", Range)(Range range)
```

Comment:
The return type is `Range` for all paths; explicit return type is better for documented functions as it provides important information to readers.